lib/experimental/fonts-library/class-wp-rest-fonts-library-controller.php

```diff
@@ -0,0 +1,216 @@
+<?php
+/**
+ * Rest Fonts Library Controller.
+ *
+ * This file contains the class for the REST API Fonts Library Controller.
+ *
+ * @package    Gutenberg
+ * @subpackage Fonts Library
+ * @since      X.X.X
+ */
+
+/**
+ * Fonts Library Controller class.
+ */
+class WP_REST_Fonts_Library_Controller extends WP_REST_Controller {
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->rest_base = 'fonts_library';
+		$this->namespace = 'wp/v2';
+	}
+
+	/**
+	 * Registers the routes for the objects of the controller.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			array(
+				array(
+					'methods'             => WP_REST_Server::EDITABLE,
+					'callback'            => array( $this, 'install_fonts' ),
+					'permission_callback' => array( $this, 'update_fonts_library_permissions_check' ),
+				),
+			)
+		);
+
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			array(
+				array(
+					'methods'             => WP_REST_Server::DELETABLE,
+					'callback'            => array( $this, 'uninstall_font_family' ),
+					'permission_callback' => array( $this, 'update_fonts_library_permissions_check' ),
+				),
+			)
+		);
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/google_fonts',
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_google_fonts' ),
+					'permission_callback' => array( $this, 'read_fonts_library_permissions_check' ),
+				),
+			)
+		);
+
+	}
+
+	/**
+	 * Removes a font family from the fonts library and all their assets.
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 */
+	function uninstall_font_family( $request ) {
```

Comment:
For all methods in this class, add the visibility keyword, e.g. `public`, `protected`, or `private`.

Reply:
Done!